changelog.d/20250226_170555_severine.bonnechere_fix_paths_exclusion_in_docker.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- Fix https://github.com/GitGuardian/ggshield/issues/548. `ggshield secret scan docker` now correctly handles ignored paths.
```

Comment:
We usually format references to issues like this (GitHub takes care of turning the issue number into a link)

```suggestion
- `ggshield secret scan docker` now correctly handles ignored paths (#548).
```